helm/cas-airflow/values.yaml

```diff
@@ -148,7 +148,7 @@ airflow:
     resources:
       limits:
         cpu: 500m
-        memory: 1.3Gi
+        memory: 1500Mi
```

Comment:
Guidance from the cluster is to not fill out limits if we can avoid it, and it seems the defaults are empty in the airflow chart https://github.com/apache/airflow/blob/c81c570ecfbd3ea6f2a42c192f6aeb967b2574d6/chart/values.yaml#L1146

Reply:
I wonder what the requests are though